index.js

```diff
@@ -1,28 +1,30 @@
 'use strict';
 
-var GetIntrinsic = require('get-intrinsic');
-var callBound = require('call-bind/callBound');
 var inspect = require('object-inspect');
+var bind = require('function-bind');
 
-var $TypeError = GetIntrinsic('%TypeError%');
-var $WeakMap = GetIntrinsic('%WeakMap%', true);
-var $Map = GetIntrinsic('%Map%', true);
+var $call = Function.prototype.call;
 
-var $weakMapGet = callBound('WeakMap.prototype.get', true);
-var $weakMapSet = callBound('WeakMap.prototype.set', true);
-var $weakMapHas = callBound('WeakMap.prototype.has', true);
-var $mapGet = callBound('Map.prototype.get', true);
-var $mapSet = callBound('Map.prototype.set', true);
-var $mapHas = callBound('Map.prototype.has', true);
+var callBound = function (original) {
+	return $call.call(bind, $call, original);
+};
```

Comment:
instead of this, can we `require('call-bind')` and pass in the intrinsics that way? what's the size impact of that?

Reply:
It would be back to square one because call-bind also uses `get-intrinsic` https://github.com/ljharb/call-bind/blob/main/index.js#L6

Reply:
ah right, good point.